CMIP6_experiment_id.json

```diff
@@ -7399,6 +7428,35 @@
             ],
             "tier":"1"
         },
+        "piControl-cmip5":{
+            "activity_id":[
+                "CMIP"
+            ],
+            "additional_allowed_model_components":[
+                "AER",
+                "CHEM",
+                "BGC"
+            ],
+            "description":"DECK: control (CMIP5-era forcing)",
+            "end_year":"",
+            "experiment":"pre-industrial control (CMIP5-era forcing)",
+            "experiment_id":"piControl-cmip5",
+            "min_number_yrs_per_sim":"500",
+            "parent_activity_id":[
+                "CMIP"
+            ],
+            "parent_experiment_id":[
+                "piControl-cmip5"
```

Comment:
Should the parent be "piControl-cmip5-spinup"? If yes, you will probably need an experiment_id entry for "piControl-cmip5-spinup".

Reply:
@slavakharin, good catch, updated PR should reflect this tweak